### ISSUE TITLE
Export metadata

### ISSUE THREAD

**Issue by 4ZUR3P47R14RCH** [photoprism/photoprism#1026]
This is my first issue on photprism. 

Is it possible to export the metadata of a photo from the web interface? I didn't manage to find

**Comment by graciousgrey** [comment]
You are right, it is not yet possible to download the metadata via the web interface. Until this is implemented, you find yml files containing metadata for each photo in the sidecar directory.
